models/safe/zkevm/safe_zkevm_matic_transfers.sql

```diff
@@ -0,0 +1,79 @@
+{{
+    config(
+        materialized='incremental',
+
+        alias = 'matic_transfers',
+        partition_by = ['block_month'],
+        unique_key = ['block_date', 'address', 'tx_hash', 'trace_address'],
+        on_schema_change='fail',
+        file_format ='delta',
+        incremental_strategy='merge',
+        post_hook = '{{ expose_spells(
+                        blockchains = \'["zkevm"]\',
+                        spell_type = "project",
+                        spell_name = "safe_matic_transfers",
```

Comment:
Sorry we may need to put project name in this field.
```suggestion
                        spell_name = "safe",
```

Reply:
yep, exactly

Reply:
the `post_hook` does nothing but tell the data explorer what to display. it won't have an impact on actual spell name. the `spell_name` argument in the `post_hook` is a bit confusing naming.

if we look at current `safe` spells on data explorer:
![image](https://github.com/duneanalytics/spellbook/assets/102681548/b296211d-0258-4e6f-820e-f37d28b1ec07)
we want to keep `safe` in this property so this new blockchain falls into this same grouping, otherwise it'll introduce a new entry in data explorer directory